docs/sql-reference/10-sql-commands/00-ddl/07-virtual-column/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: VIRTUAL COLUMN
+---
+import IndexOverviewList from '@site/src/components/IndexOverviewList';
+import EEFeature from '@site/src/components/EEFeature';
+
+<EEFeature featureName='VIRTUAL COLUMN'/>
+
+A virtual column is a construct formed by extracting nested fields within [Variant](../../../00-sql-reference/10-data-types/43-data-type-variant.md) data and storing that data in separate storage files. Virtual columns offer enhanced efficiency in querying Variant data, with the following key benefits:
+
+- **Accelerated Query Processing**: Virtual columns streamline the querying process by eliminating the need to traverse the entire nested structure to locate the desired data. Direct data retrieval from virtual columns parallels the process of accessing regular columns, resulting in a significant acceleration of query execution.
```

Comment:
Better to add 'when will use virtual column' part, say this case:
If the variant columns have many data already, then we create the virtual columns, when will use the new virtual column? The old data will not(from explain), new data will.

Reply:
@b41sh Can u help clarify about this scenario? Thanks

Reply:
If a table already has a lot of variant data stored in it, just creating a virtual column is not enough to make it effective, we need to execute `refresh` after creating it to generate real virtual column data, and similarly, we need to execute `refresh` after altering virtual columns to generate a new virtual column data. We need to add some notes to clarify this so that users don't think that just creating or altering virtual columns is enough!